code/processes/reporter.q

```diff
@@ -115,10 +115,13 @@ runreport:{[tab]
 		.timer.once[startts;fp;"Reporter - ",string tab`name];
 		`..timerids upsert 1!select id,periodend:nextrun from .timer.timer where fp~/:funcparam;
 	:()];
-	/ - if the current time is within the start and end timestamps, use current time as start time, else use the startts
+        / - if timezone is specified convert the start and end timestamps to the time used by the system 
+    startts:?[not null tab[`timezone];.tz.ttz[`GMT;tab[`timezone];$[.proc.localtime;gtime[startts];startts]];startts];
+    endts:?[not null tab[`timezone];.tz.ttz[`GMT;tab[`timezone];$[.proc.localtime;gtime[endts];endts]];endts];
```

Comment:
can you align the indentation of these lines with the surrounding please?

Reply:
It was on my VSCode, not sure what the issue was when I pushed. Fixed now